Cargo.lock

```diff
@@ -226,19 +227,19 @@ dependencies = [
 
 [[package]]
 name = "clap"
-version = "4.4.6"
+version = "4.5.20"
```

Comment:
I hope it's ok that adding `clap_complete` updated the `clap` dependency. I could also downgrade `clap_complete` so it uses `clap` `4.4.6`.

Reply:
I downgraded as the newer version is not compatible with this crates MSRV `1.70`.